Bottom: 17591f73e08babf97ed8dd45f8b91586f202332d
Top:    f07fc364bd622c6c2c759e433844f08d5161f151
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2013-07-20 10:21:59 -0400

Refresh of asiarf

---

diff --git a/target/linux/ramips/base-files/etc/hotplug.d/firmware/10-rt2x00-eeprom b/target/linux/ramips/base-files/etc/hotplug.d/firmware/10-rt2x00-eeprom
index 31e75da..9293a29 100644
--- a/target/linux/ramips/base-files/etc/hotplug.d/firmware/10-rt2x00-eeprom
+++ b/target/linux/ramips/base-files/etc/hotplug.d/firmware/10-rt2x00-eeprom
@@ -61,7 +61,7 @@ case "$FIRMWARE" in
 	all5002 | \
 	all5003 | \
 	argus-atp52b | \
-	awm002 | \
+	awm002-evb | \
 	bc2 | \
 	br6425 | \
 	broadway | \
diff --git a/target/linux/ramips/base-files/lib/ramips.sh b/target/linux/ramips/base-files/lib/ramips.sh
index d9ee907..1f81b24 100755
--- a/target/linux/ramips/base-files/lib/ramips.sh
+++ b/target/linux/ramips/base-files/lib/ramips.sh
@@ -46,8 +46,8 @@ ramips_board_detect() {
 	*"Argus ATP-52B")
 		name="argus-atp52b"
 		;;
-	*"AsiaRF AWM002")
-		name="awm002"
+	*"AsiaRF AWM002-EVB")
+		name="awm002-evb"
 		;;
 	*"BR6524N")
 		name="br6524n"
